### ISSUE TITLE
Support for apt

### ISSUE THREAD

**Issue by geetayadav** [mirror/busybox#8]
Please provide support for apt-get or some other package installer.

**Comment by geonaumov** [comment]
Yes apt will be nice but it's too complicated. Maybe just a simplified version that checks for newer packages from a http server, so that the repo can also be hosted with busybox only. I will try to script something like that in shell.

**Comment by Saijin-Naib** [comment]
Would APK from Alpine make sense?
